Add script to auto-upload all reports to the right google sheet (#28)
mozilla/ecosystem-test-scripts: docs/metric-update-procedure/README.md

The sheets are typically updated on Monday mornings (North America ET/PT) to ensure the values are
available for team check-in meetings. The process of updating all three Google Sheets can take up
to 10 minutes.

## First set up (just has to be done once)

### Prepare the auto-uploader to Google Sheets

> #### Step 1: Create a Google Cloud Project
> * Go to the [Google Cloud Console](https://console.cloud.google.com/apis/dashboard?project=ecosystem-test-eng).
> * Create a new project or select an existing one.

Reviewer: nit:

```suggestion
#### Step 1:Navigate to the ETE Google Cloud Project
* Go to the [Google Cloud Console](https://console.cloud.google.com/apis/dashboard?project=ecosystem-test-eng).
```

## 3. Import CSVs to Google Spreadsheets

To import the generated CSVs into the 3 different Google Spreadsheets, execute the following make command:

```shell
make run_google_sheet_uploader
```

This will go through the `reports` folder and takes every CSV file and imports them into the right spreadsheet and tab. This will happen based on the mapping in the `config.ini` file.

- If the report being imported is a results or coverage report. Convert the type of the date

>      column to 'plain text' so that the graphs display at an even cadence.

Author: Does this still need to happen? As well as `3.3 Update Trend Table Dates`? If so, we can probably add this to the script as well.

Author: I think we can add this once the basic uploading works as expected and is merged into `main`.
